Added maxWidth property to Image

Romster opened the pull request:
Hello! I use this library for inserting images (qr-codes), generated by third-party, into document. And I was a bit frustrated that I can't affect on image size in document. 
Since version 3.3.0 _docx4j_ library provides method
`createImageInline(String filenameHint, String altText, int id1, int id2, boolean link, int maxWidth)`
From the official documentation: this parameter is 
> Useful if the image is to be inserted into a table cell of known width.

So I added maxWidth property into _Image_ object and wrote a code which handle it in _ImageResolver_.
My changes have no side effects, so people can use Image class as they did before.

thombergs commented:
Thanks for the contribution!

Before merging this PR, I would like to understand the max image width property. What is the unit of maxWidth? I would like to add some Javadoc to explain it.

Romster commented:
Thank you for this library! :)
The twip is the unit. Got it from the docx4j source code.
CxCy#scale:
`writableWidthTwips = maxWidth;`

Romster commented:
I've added basic docs for Image class.
Also I've removed setMaxWidth method because I don't think we need this one: there are different ways to work with images in docx4j, so I think it would be better to create factory methods for Image in the future (without sizes, with specified width only, with heights and width, with maxWidth).
